Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```
@@ -687,4 +687,10 @@ tests:
 - Splunk-Test_default_handler
 - SplunkPy_KV_commands_default_handler
 - SplunkPy parse-raw - Test
+- Splunk-Test_requests_handler
+- SplunkPy_KV_commands_requests_handler
+- SplunkPy-Test-V2_requests_handler
+- SplunkPySearch_Test_requests_handler
+- SplunkPy_KV_commands
+- playbook-SplunkPySearch_Test
```

```suggestion
- SplunkPy_KV_commands
- playbook-SplunkPySearch_Test
```
remove playbooks that are not in conf.json